kubernetes/ansible/roles/sunbird-monitoring/templates/alertrules.yaml

```diff
@@ -84,3 +84,13 @@ secor_process_names:
 
 redis_process_names:
   {{ redis_process_names | to_yaml | indent( width=2) }}
+
+service_health_checks:
```

Comment:
define this in roles/defaults/main.yaml and add
service_health_checks: {{ service_health_checks | to_yaml }}

Reply:
Updated as requested.